kuma/static/js/wiki-compat-tables.js

```diff
@@ -13,9 +13,50 @@
     // is it a browser compat section?
     if(sectionHeadIsMatch) {
         // come back down DOM hiding things until we reach the comapt table.
-        $sectionHead.nextUntil('.bc-api').hide();
+        $sectionHead.nextUntil('.bc-api').hide().addClass('bc-old-hide');
     }
 
+    // add beta notice & associated functonality
+    var $betaMenuWrapper = $('<div>', {class: 'bc-beta-menu'});
+    var $betaMenuTrigger = $('<a>', {text: 'New compatibility tables are in beta ', href: '/en-US/docs/New_Compatability_Tables_Beta'}).append($('<i>', {class: 'icon-caret-down', 'aria-hidden': 'true'}));
+    var $betaSubmenu = $('<ul>', {class: 'submenu js-submenu'});
+    var betaSubmenuItems = [];
+
+    var $betaLink = $('<a>', {text: 'More about the beta.', href: '/en-US/docs/New_Compatability_Tables_Beta'});
+    betaSubmenuItems.push($betaLink);
+
+    var $betaSurvey = $('<a>', {text: 'Take the survey', href: 'http://www.surveygizmo.com/s3/2342437/0b5ff6b6b8f6', class: 'external external-icon'});
```

Comment:
Should we consider opening this in a new window?  New site, and so they can flip back and look at it when they're answering?